internal/web/templates/pages/repository.html

```diff
@@ -0,0 +1,39 @@
+{{ define "content" }}
+  <div class="container mx-auto p-4 shadow-md rounded-lg mt-8">
+    <h1 class="text-2xl font-bold mb-4">Repository Configuration</h1>
+    <form class="space-y-4">
+      <div>
+        <label for="name" class="form-label">Name</label>
+        <div class="input-group mb-2">
+          <input type="text" id="name" name="name" value="{{ .Repository.Name }}" disabled class="form-control" />
+        </div>
+      </div>
+      <div>
+        <label for="url" class="form-label">URL</label>
+        <div class="input-group mb-2">
+          <input type="text" id="url" name="url" value="{{ .Repository.Spec.Url }}" required class="form-control" />
+        </div>
+      </div>
+      <div>
+        <label for="default" class="form-check mt-1 mb-3">
+          <input
+            type="checkbox"
+            id="default"
+            name="default"
+            {{ if .Repository.IsDefaultRepository }}checked{{ end }}
+            class="form-check-input" />
```

Comment:
Unchecking this for the default repo does nothing, which I think is OK. There is not supposed to be no default repo. But I think it would make sense to also disable the checkbox for the current default repo, so that it can not be unchecked.

Reply:
done